.forge-snapshots/add liquidity.snap

```diff
@@ -0,0 +1 @@
+466644
```

Comment:
maybe follow similar naming conventions in repo for gas tests. i.e. `FullRangeAddLiquidity`. (I Know it's diff than core, I guess were still figuring this out but might be nice to have periphery consistent)